server/handler/HookHandler.js

```diff
@@ -48,12 +48,13 @@ class HookHandler {
       if (repo.checks.length) {
         const zapprFileContent = await this.githubService.readZapprFile(owner.login, name, repo.checks[0].token)
         const zapprfile = new ZapprConfiguration(zapprFileContent)
-        config = zapprfile.isValid() ? zapprfile.getConfiguration() : config
+        config = zapprfile.getConfiguration()
       }
 
       if (Specification.isTriggeredBy(event)) {
-        const token = await getToken(repo, Specification.TYPE)
-        await this.specification.execute(config, payload, token)
+        getToken(repo, Specification.TYPE).then(token =>
```

Comment:
Semantically almost nothing and I admit I forgot to revert this. Difference is we're not blocking until specification check is executed before we report back to Github.